guides/common/modules/ref_inclusion-of-remote-scap-resources.adoc

```diff
@@ -0,0 +1,41 @@
+[id="inclusion-of-remote-scap-resources_{context}"]
+= Inclusion of Remote SCAP Resources
+
+SCAP data streams can reference remote resources, such as OVAL files, that the SCAP client fetches over the internet when it runs on hosts.
+If a data stream requires a remote resource, you can see a warning from the OpenSCAP Scanner tool on your {ProjectServer}, such as:
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# oscap info _/usr/share/xml/scap/ssg/content/ssg-rhel8-ds.xml_ | grep "WARNING"
+WARNING: Datastream component 'scap_org.open-scap_cref_security-data-oval-com.redhat.rhsa-RHEL8.xml.bz2'
+points out to the remote 'https://access.redhat.com/security/data/oval/com.redhat.rhsa-RHEL8.xml.bz2'.
+Use '--fetch-remote-resources' option to download it.
+WARNING: Skipping 'https://access.redhat.com/security/data/oval/com.redhat.rhsa-RHEL8.xml.bz2' file
+which is referenced from datastream
+----
+
+By default, the SCAP client is configured to ignore the remote resources, which causes the XCCDF rules that rely on the resources to be skipped and result in the `notchecked` status.
+In a connected network environment, you can enable the download of remote resources on hosts in {Project}.
+
+Using the Ansible deployment method::
+Override the following Ansible variable:
+
+* Name: `foreman_scap_client_fetch_remote_resources`
+* Type: `boolean`
+* Value: `true`
+
++
+For more information, see {ManagingConfigurationsAnsibleDocURL}Overriding_Ansible_Variables_in_{project-context}_ansible[Overriding Ansible Variables in {Project}] in _{ManagingConfigurationsAnsibleDocTitle}_.
+
+Using the Puppet deployment method::
+Configure the following Puppet Smart Class Parameter:
+
+* Name: `fetch_remote_resources`
+* Type: `boolean`
+* Value: `true`
+
++
+For more information, see {ManagingConfigurationsPuppetDocURL}Configuring_Puppet_Smart_Class_Parameters_managing-configurations-puppet[Configuring Puppet Smart Class Parameters] in _{ManagingConfigurationsPuppetDocTitle}_.
+
+ifdef::satellite[]
+If you require remote resources in a disconnected network environment, see ... (xref)
```

Comment:
This looks unfinished.

Reply:
I'll remove it here and add it in another PR.